modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/Apache2Confgen.java

```diff
@@ -0,0 +1,106 @@
+package io.swagger.codegen.languages;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+
+import io.swagger.codegen.CliOption;
+import io.swagger.codegen.CodegenConfig;
+import io.swagger.codegen.CodegenConstants;
+import io.swagger.codegen.CodegenOperation;
+import io.swagger.codegen.CodegenType;
+import io.swagger.codegen.DefaultCodegen;
+import io.swagger.codegen.SupportingFile;
+
+public class Apache2Confgen extends DefaultCodegen implements CodegenConfig {
```

Comment:
What about naming the class as "Apache2ConfigCodegen" to make it consistent with other generators (clients, servers, documentations)?